### PR TITLE
L10n fr

### PR DESCRIPTION
**Pull request by tdd**
Hi Rod, me again (yeah, I'm full steam on this this week…) :smile: 

So here's the fr L10n for learnyounode; I updated the deps to use workshopper 2.3.1 and workshopper-exercise 2.3.0.  This had boilerplate in its deps but didn't use it, so I stripped it.

FYI: I had originally translated the menu items in title case (or thereabouts), but due to the outstanding issue with terminal-menu / visualwidth, I had to put it back to full uppercase so diacritics wouldn't offset "[COMPLETE]" markers.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -38,9 +39,8 @@
     "through": "~2.3.6",
     "through2": "~0.6.3",
     "through2-map": "~1.4.0",
-    "workshopper": "~2.3.0",
-    "workshopper-boilerplate": "~0.0.1",
```
Comment (rvagg) [inline package.json]: good catch .. looks like this has been hiding in here for quite a while

**Comment by rvagg** [comment]
I'd really like to be able to add >1 person to manage fr and get their sign-off here on the translation as well. Can you try and convince someone who's interesting in helping maintaining, answering issues, double-checking work that the rest of us can't read, etc.?

**Comment by rvagg** [comment]
bed time here, will try and get this sorted out tomorrow, hopefully you can find someone else to get on board by then as well

**Comment by tdd** [comment]
Hey Rod, just like for other repos, I'll ping @JeremiePat to see if he'd like to get on board with this.

**Comment by ralphtheninja** [comment]
@gorhgorh might be able to help out as well :)

**Comment by tdd** [comment]
True :smile: he's on board with NodeSchool Paris anyway!

**Comment by gorhgorh** [comment]
true :) and I would be glad to help

**Comment by tdd** [comment]
Excellent! Can you spare a moment to proofread the L10n fr in this PR, and +1 it if you find it alright? This way we can merge and bump version :smile:

Thanks a lot!

File: exercises/http_client/problem.fr.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+peut être l’URL que vous voulez récupérer ; passez une fonction de rappel
+en deuxième argument.
+
+Contrairement aux autres fonctions de rappel, celle-ci a comme signature :
+
+```js
+function callback (response) { /* ... */ }
+```
+
+…dans laquelle l’objet `response` est un objet **flux** Node.  Vous pouvez
```
Comment (gorhgorh) [inline exercises/http_client/problem.fr.md]: I'm not sure "stream" should translated, or maybe something like "**flux** (stream)" for the first occurrence ?

**Comment by gorhgorh** [comment]
check done, all good :shipit: ! 

Good work to all on the L10n effort, I'll updates the nodebots ones asap :)

side note : my current terminal color scheme (solarized dark) is pretty popular, and comments from the problems are the same color as the scheme bg, hiding them, I'll investigate this properly later.

**Comment by tdd** [comment]
Alright then, thanks Jérôme!
@rvagg now we're just waiting for your merge and bump :wink:

**Comment by rvagg** [comment]
published as v2.5.0, thanks a ton @tdd and it's a pleasure to have you on board!

**Comment by tdd** [comment]
My pleasure :smile:

We're hard at work on other workshopper-based ones right now, such as expressworks, promise-it-wont-hurt, async-you, makemehapi, count-to-6, etc.

I'm looking at your latest comment on boilerplate just now so we can get this shipped ASAP.

**Pull request merged**